cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/cinema/user/", include("user.urls", namespace="user")),
```

Comment:
The `include` function for the `user.urls` module specifies a `namespace` argument. Ensure that the `user.urls` module has an `app_name` attribute defined to match this namespace, otherwise, it will raise an error.